src/components/Avatar.js

```diff
@@ -2,8 +2,18 @@
 // returns an img element with className 'Avatar'
 // img element should have src and alt attributes
 
-export default function Avatar(props) {    
-    const { person } = props;
+import { keyboard } from "@testing-library/user-event/dist/keyboard";
```

Comment:
Importing keyboard here is wrong, if you want to do further testing you should do it in the test files 
path:  src/App.test.js